components/script_bindings/webidls/CanvasRenderingContext2D.webidl

```diff
@@ -287,6 +287,7 @@ interface Path2D {
   constructor();
   constructor(Path2D other);
   constructor(DOMString pathString);
-  undefined addPath(Path2D path/*, SVGMatrix? transformation*/);
+  [Throws]
+  undefined addPath(Path2D path, optional DOMMatrix2DInit transform = {});
```

Comment:
I know this is technically the correct definition for this but should there also be another definition like
```
undefined addPath(Path2D path);
```
with an implementation that skips applying the transformation on the `PathSegments`? Or should I skip applying the matrix if it's the identity matrix?

Reply:
Honestly, I would just keep multiplying identity matrix for simplicity.